cmd/exporter/exporter.go

```diff
@@ -100,13 +99,15 @@ func runServer(ctx context.Context, cfg *config.Config, csp provider.Provider) e
 	errChan := make(chan error)
 
 	go func() {
-		log.Printf("Listening on %s%s", cfg.Server.Address, cfg.Server.Path)
+		log.LogAttrs(ctx, slog.LevelInfo, "Starting server",
+			slog.String("address", cfg.Server.Address),
+			slog.String("path", cfg.Server.Path))
 		errChan <- server.ListenAndServe()
 	}()
 
 	select {
 	case <-ctx.Done():
-		log.Print("shutting down server")
+		log.LogAttrs(ctx, slog.LevelInfo, "Shutting down server")
```

Comment:
oops - this should actually be `log`, but the arg should be passed into the `runServer` function (see above suggestion)